presto-main/src/main/java/com/facebook/presto/server/TaskResource.java

```diff
@@ -324,6 +324,17 @@ public void abortResults(@PathParam("taskId") TaskId taskId, @PathParam("bufferI
         taskManager.abortTaskResults(taskId, bufferId);
     }
 
+    @DELETE
+    @Path("{taskId}/remote-source/{sourceTaskId}")
+    @Produces(APPLICATION_JSON)
+    public void removeRemoteSource(@PathParam("taskId") TaskId taskId, @PathParam("sourceTaskId") TaskId sourceTaskId, @Context UriInfo uriInfo)
```

Comment:
`uriInfo` is unused.

Reply:
Interesting... this means `abortResults` (path is `{taskId}/results/{bufferId}`) also have this redundant parameter :)